aw-models/src/key_value.rs

```diff
@@ -0,0 +1,27 @@
+use chrono::{DateTime, Utc};
+
+#[derive(Serialize, Deserialize)]
+pub struct Key {
+    pub key: String
+}
+
+// TODO: Invent a better naming scheme than calling the non-timestampy one "KV"
```

Comment:
The simple solution would be to wrap the timestamp in the KeyValue in an Option

Reply:
Fixed in 37e7623